### ISSUE TITLE
Image.java:37-38: Continue implementing the operations...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#96]
The puzzle `86-390ec7ae` from #86 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/68ad57f1a8ee7cde78b455fa65efff79f5e63446/src/main/java/com/amihaiemil/docker/Image.java#L37-L38

The puzzle was created by amihaiemil on 26-Apr-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ab56d234-9c5a-4a94-9bc9-5e54c14be93d, thread: BkParallel-5
com.mongodb.MongoCollectionImpl[558] com.mongodb.MongoWriteException: quota exceeded
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/G6LPQQV2P) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#96](https://github.com/amihaiemil/docker-java-api/issues/96) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ab56d234-9c5a-4a94-9bc9-5e54c14be93d, thread: BkParallel-2
com.mongodb.MongoCollectionImpl[558] com.mongodb.MongoWriteException: quota exceeded
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ab56d234-9c5a-4a94-9bc9-5e54c14be93d, thread: BkParallel-6
com.mongodb.MongoCollectionImpl[558] com.mongodb.MongoWriteException: quota exceeded
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ab56d234-9c5a-4a94-9bc9-5e54c14be93d, thread: BkParallel-9
com.mongodb.MongoCollectionImpl[558] com.mongodb.MongoWriteException: quota exceeded
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)

**Comment by 0crat** [comment]
The job [#96](https://github.com/amihaiemil/docker-java-api/issues/96) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1271828049); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@0crat waiting on review for #101

**Comment by 0crat** [comment]
> @0crat waiting on review for #101 ([here](https://github.com/amihaiemil/docker-java-api/issues/96#issuecomment-387499854))

@llorllale The impediment for [#96](https://github.com/amihaiemil/docker-java-api/issues/96) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `86-390ec7ae` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#103](https://github.com/amihaiemil/docker-java-api/issues/103) is still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#96](https://github.com/amihaiemil/docker-java-api/issues/96) is now out of scope
